.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
Didn't created .gitignore and pushed the `node_modules` into code repo. Which is wrong practice.